### PR TITLE
Fixes #7117: Use the webhelp doc as embedded doc in the webapp

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7117

### CODE REVIEW

**Comment by amousset** [comment]
Should not be merged before #740.

File: rudder-webapp/SOURCES/Makefile [hunk 1]
```diff
@@ -47,10 +47,9 @@ localdepends: ./rudder-sources ./maven2/bin/mvn ./rudder-users.xml ./rudder-doc
 
 ./rudder-doc: /usr/bin/wget
 	# Get user manual from rudder-project.org and rename directory containing it to rudder-doc
-	$(WGET) -P ./rudder-doc/html -r -np -nH --cut-dirs=1 http://www.rudder-project.org/rudder-doc-${RUDDER_MAJOR_VERSION}/rudder-doc.html
-	mkdir -p ./rudder-doc/pdf
-	# Move the generated PDF from html directory to pdf
-	mv ./rudder-doc/html/rudder-doc.pdf ./rudder-doc/pdf/
+	$(WGET) -P ./rudder-doc/html -r -np -nH --cut-dirs=1 http://www.rudder-project.org/doc-${RUDDER_MAJOR_VERSION}/index.html
```
Comment (jooooooon) [inline rudder-webapp/SOURCES/Makefile]: Please don't include the index.html at the end of the URL. I'd rather use a standard URL and let the webserver serve whatever it thinks (aka what we configure) is best.

**Comment by amousset** [comment]
PR updated.

File: rudder-webapp/SOURCES/Makefile [hunk 2]
```diff
@@ -47,10 +47,9 @@ localdepends: ./rudder-sources ./maven2/bin/mvn ./rudder-users.xml ./rudder-doc
 
 ./rudder-doc: /usr/bin/wget
 	# Get user manual from rudder-project.org and rename directory containing it to rudder-doc
-	$(WGET) -P ./rudder-doc/html -r -np -nH --cut-dirs=1 http://www.rudder-project.org/rudder-doc-${RUDDER_MAJOR_VERSION}/rudder-doc.html
-	mkdir -p ./rudder-doc/pdf
-	# Move the generated PDF from html directory to pdf
-	mv ./rudder-doc/html/rudder-doc.pdf ./rudder-doc/pdf/
+	$(WGET) -P ./rudder-doc/html -r -np -nH --cut-dirs=1 http://www.rudder-project.org/doc-${RUDDER_MAJOR_VERSION}/
+	# Download the pdf version of the manual
+	$(WGET) -P ./rudder-doc/pdf 1 http://www.rudder-project.org/rudder-doc-${RUDDER_MAJOR_VERSION}/rudder-doc.pdf
```
Comment (jooooooon) [inline rudder-webapp/SOURCES/Makefile]: The "1" here looks like a typo.

**Comment by amousset** [comment]
PR updated.

**Comment by jooooooon** [comment]
This looks good now, but I don't think we should make the change on the 2.10 branch. This is not a bug fix, so does not fit our policy for changes on maintenance releases. Also, it would be a very surprising change in user experience for any users used to consulting that documentation, that suddenly after a minor version upgrade, changes completely.

Please retarget to master unless you see a reason I'm missing here.

**Comment by amousset** [comment]
Sure, retargeting.

**Pull request closed without merging**